### PR TITLE
Support for xUnit.net

### PR DESCRIPTION
**Pull request by tischda**
I was trying to setup a GO projet on appveyor.com, but was confused by their support for xUnit, which is in fact `xUnit.net` - the format is described here:

https://xunit.codeplex.com/wikipage?title=XmlFormat

This is my attempt to implement it in go2xunit.

### CODE REVIEW

**Comment by tebeka** [comment]
Thanks! I'm currently out of the country but will have a look next week.

File: .hgignore [hunk 1]
```diff
@@ -1,8 +0,0 @@
-syntax: glob
```
Comment (tebeka) [inline .hgignore]: Please don't remove mercurial files. The main development of `go2xunit` is done [on bitbucket](https://bitbucket.org/tebeka/go2xunit) and github is a mirror for users who prefer it.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,28 @@
-# go2xunit
+# go2xunit [![Build Status](https://travis-ci.org/tischda/go2xunit.svg)](https://travis-ci.org/tischda/go2xunit)
 
 Converts `go test -v` (or `gocheck -vv`) output to xunit compatible XML output
 (used in [Jenkins][jenkins]/[Hudson][hudson]).
 
+This is a fork from go2xunit by Miki Tebeka <miki.tebeka@gmail.com>.
```
Comment (tebeka) [inline README.md]: I'll remove these when merging :)

Comment (tischda) [inline README.md]: Oh sorry, I have not read the documentation properly: "After your pull
request is sent, any new commits pushed to your branch will
automatically be added to the pull request.".

The pull request should have ended here:

commit 253d2d5b
Date:   Sat Jun 6 00:36:17 2015 +0200

    set assembly attribute

After that, I ripped everything to pieces to play around with your code
(I'm learning GO).
Do you want me to create another pull request ?

Comment (tebeka) [inline README.md]: If you can it'll be great.

I *do* plan to do some refactoring, but that's not related.

**Comment by tischda** [comment]
ok, I will create a new request.

**Pull request closed without merging**